setup/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
      - ../data/ipfs/data:/data/ipfs
      - ../data/ipfs/staging:/export
   testrpc:
-    image: "harshjv/testrpc:latest"
+    image: "testprc_persist:latest"
```

Comment:
It is a local docker image that I played around with @TheOriginalAlex to get the Sonar contract already bootstrapped into `testrpc` (`prc` is a typo that I noticed to late). Was hoping we could automate the build process by now but it seems to be a tad complicated. Switching back to the previous image to have a somewhat working branch!